index.js

```diff
@@ -8,13 +5,20 @@ var zlib = require('zlib')
 var urlUtils = require('url')
 
 /**
- * A feature service object needs a url and some optional params to make requests
- * this module exposes a pageQueue that can take an array of page url
- * the pageQueue will report back each page of features as they return
- * @module FeatureService
+ * Feature Service constructor. Requires a URL.
+ * Exposes a pageQueue that can take an array of page URLs.
+ * The pageQueue will report back each page of features as they return.
  *
+ * @class
+ * @param {string} url - address of feature service
+ * @param {object} options - layer (default: 0)
  */
 var FeatureService = function (url, options) {
+  // catch omission of `new` keyword
+  if (!(this instanceof FeatureService)) {
+    return new FeatureService(url, options)
+  }
```

Comment:
This will ensure omitting `new` doesn't cause variables leaking into the global `this` object.